src/eq_schema/RoutingRule.js

```diff
@@ -0,0 +1,80 @@
+const mapFields = require("../utils/mapFields");
+const mapping = {
+  Equal: "equals",
+  NotEqual: "not equals"
+};
+const toRunner = mapFields(mapping);
+
+const {
+  isNil,
+  filter,
+  flatMap,
+  flatMapDeep,
+  flatten,
+  pick,
+  get,
+  findIndex
+} = require("lodash");
+
+class RoutingRule {
+  constructor(rule, pageId, ctx) {
+    this.goto = {};
+    if (!rule.goto) {
+      this.goto.block = this.getUndefinedGoto(pageId, ctx);
+    } else if (rule.goto.__typename === "QuestionPage") {
+      this.goto.block = this.getGotoBlock(rule.goto);
+    } else {
```

Comment:
I'd make this an `else if` and explicitly check for `Section`. We should probably bail out of the process or Error if something other than QuestionPage or Section is passed down.